docs/api.md

```diff
@@ -0,0 +1,40 @@
+Short API doc.
+
+# Authentication
+* POST /api/login - basic auth to get token
+* POST /api/logout - close session
+* GET /api/token - verifies whether auth token is still valid 
+
+# Users
```

Comment:
Maybe create separate methods for user administration?
For instance, give or remove permissions to manage teams.